### ISSUE TITLE
hpAnyKeyPrmpt not enabled on V6 Core

### ISSUE THREAD

**Issue by antoinechauvn** [rconfig/rconfig#94]
**Describe the bug**
Based on source code provided on github, it seems like the hpAnyKeyPrmpt function was not enabled:

```
    // private function HPChecks()
    // {
    //     if ($this->_hpAnyKeyStatus === true) {
    //         $this->connectionObj->connection->read($this->_hpAnyKeyPrmpt, SSH2::READ_REGEX);
    //         $this->connectionObj->connection->write("\n");
    //     }
    //     $this->connectionObj->connection->read('~'.$this->connectionObj->devicePrompt.'~', SSH2::READ_REGEX);
    // }
```

**To Reproduce**
1. Create a Template
2. Use hpAnyKeyPrmpt
3. Try to debug the device using CLI commands

**Expected behavior**
My equipment is a digi IX20. By default, these devices display a menu for which you simply press enter to exit the menu.

**Screenshots**
![image](https://github.com/rconfig/rconfig/assets/83721477/31d28de3-e1f2-49ff-8f5b-d7db6394a39b)

**rConfig Version Information (You can copy this from the rConfig dashboard):**
OS VERSION
Red Hat Enterprise 8.8 (Ootpa)
PHP / LARAVEL VERSION
8.2.12 / 10.32.1
REDIS VERSION
5.0.3
MYSQL VERSION
10.3.39

**Logs info (can be retrieved from /var/www/html/rconfig6/storage/logs - try to paste relevant logs only)
Horizon started successfully.
  2023-12-18 08:50:07 App\Jobs\DownloadConfigNow ..................... RUNNING
  2023-12-18 08:51:10 App\Jobs\DownloadConfigNow .................. 1m 2s DONE

**Additional context**

```yml
# rConfig connection template - DO NOT EDIT DIRECTLY
## Notes: 
    ## Remember to update permissions for the templates folder after uploading new template files.
    ## run 'chown -R apache /home/rconfig' on your rconfig server CLI
    ## all items below that contain free text should be contained within quotation marks " "
    ## For new community submitted templates visit: https://github.com/rconfig/rConfig-templates

main:
  name: "DiGi IX20 - SSH - No Enable"
  desc: "DiGi IX20 SSH based connection without enable mode"
connect:
  # connection timeout in seconds
  timeout: 30
  # type either telnet or ssh - must be in lowercase
  protocol: ssh
  # type port number for connection protocol choose from 1 - 65535
  port: 22
auth:
  # Set username name prompt. Must be in quotes. If left blank, username will not be sent.
  username: ''
  # Set password name prompt. Must be in quotes
  password: 'Password:'
  # Set enable mode on or off
  enable: off
  # set enable mode command. Must be in quotes
  enableCmd: 'enable'
  # set enable mode password prompt. Must be in quotes
  enablePassPrmpt: ''
  # set HP 'press any key' status. Is 'on' or 'off'
  hpAnyKeyStatus: off
  # set HP 'press any key' prompt.
  hpAnyKeyPrmpt: '* Use of the network is restricted to authorized users. *'
config:
  # Experimental: Some vendors use different linebreak settings. Choose 'n' (default) or 'r'
  linebreak: 'n'
  # Disable config scrolling/ paging command. on/off
  paging: off
  # Disable config scrolling/ paging command. Leave blank, or set as required. Must place command in quotations
  pagingCmd: ''
  # re-enable config scrolling/ paging command. Leave blank, or set as required. Must place command in quotations
  resetPagingCmd: ''
  # Set pager prompt. Must place command in quotations
  pagerPrompt: ''
  # Set pager prompt key-stroke. Must place command in quotations
  pagerPromptCmd: ' '
  # Save configuration command. Leave blank, or set as required. Must be in quotes
  saveConfig: ''
  # Set exit command. Must place command in quotations
  exitCmd: 'exit'
options:
  AnsiHost: 'yes'
```

**Comment by stephenstack** [comment]
Hi,

This is old legacy code for really old HP/ 3Com switches. If this works for your digi IX20s, i am happy to reenable it, as it is switched from the device connection template. Do you want to PR it? or shall it make the code change?

**Comment by antoinechauvn** [comment]
Hello,

Exactly, it doesn't work for me. 

For your information, when I log in, the menu appears and then I have to choose admin (or press enter, it's the same thing). Then I can execute the commands (basic command in Rconfig).

When does hpAnyKeyPrmpt run? Before or after the commands? Because I don't see it being executed. I'll have a look at the source code and let you know what happens.

**Comment by stephenstack** [comment]
Can you send a screenshot of the initial login process here. It will be difficult to test without access to one of these devices. We may do a remote troubleshooting session. If this is required, send a quick email to info@rconfig.com

**Comment by stephenstack** [comment]
Was this resolved? Can we close this issue?

**Comment by antoinechauvn** [comment]
Hello,

I've sent you a link to an invitation to a meeting.

The problem persists. The same applies to our F5 equipment when a menu appears.

Would you be available this week?

**Comment by stephenstack** [comment]
resolved with #97
